build/Dockerfile

```diff
@@ -19,11 +19,11 @@ RUN adduser --comment '' fdb && chown fdb /opt
 
 # wget of bintray without forcing UTF-8 encoding results in 403 Forbidden
 RUN cd /opt/ &&\
-    curl -L https://dl.bintray.com/boostorg/release/1.67.0/source/boost_1_67_0.tar.bz2 > boost_1_67_0.tar.bz2 &&\
-		echo "2684c972994ee57fc5632e03bf044746f6eb45d4920c343937a465fd67a5adba  boost_1_67_0.tar.bz2" > boost-sha.txt &&\
+    curl -L https://dl.bintray.com/boostorg/release/1.72.0/source/boost_1_72_0.tar.bz2 > boost_1_72_0.tar.bz2 &&\
+		echo "2684c972994ee57fc5632e03bf044746f6eb45d4920c343937a465fd67a5adba  boost_1_72_0.tar.bz2" > boost-sha.txt &&\
 		sha256sum -c boost-sha.txt &&\
-    tar -xjf boost_1_67_0.tar.bz2 &&\
-    rm -rf boost_1_67_0.tar.bz2 boost-sha.txt boost_1_67_0/libs
+    tar -xjf boost_1_72_0.tar.bz2 &&\
+    rm -rf boost_1_72_0.tar.bz2 boost-sha.txt boost_1_72_0/libs
```

Comment:
I confirmed you can just delete the changes to build/Docker.  They conflict, and master has changes that supersede this.

Reply:
@sears Because the same build Docker is often used to build previous versions of the software, I would like to not remove boost 1.67 from the build docker at this time in order to give those with previous versions the ability to build with the same docker.

Reply:
Agreed.  master's build/Dockerfile currently installs 1_67_0 and 1_72_0.  (Due to Alvin's commit:  aa6dfc646cefa413cb8780b025d771e1b79abfd5)

```patch
@@ -19,11 +18,16 @@ RUN adduser --comment '' fdb && chown fdb /opt
 
 # wget of bintray without forcing UTF-8 encoding results in 403 Forbidden
 RUN cd /opt/ &&\
-    curl -L https://dl.bintray.com/boostorg/release/1.67.0/source/boost_1_67_0.tar.bz2 > boost_1_67_0.tar.bz2 &&\
-               echo "2684c972994ee57fc5632e03bf044746f6eb45d4920c343937a465fd67a5adba  boost_1_67_0.tar.bz2" > boost-sha.txt &&\
-               sha256sum -c boost-sha.txt &&\
+    curl -L https://dl.bintray.com/boostorg/release/1.67.0/source/boost_1_67_0.tar.bz2 -o boost_1_67_0.tar.bz2 &&\
+    echo "2684c972994ee57fc5632e03bf044746f6eb45d4920c343937a465fd67a5adba  boost_1_67_0.tar.bz2" > boost-sha-67.txt &&\
+    sha256sum -c boost-sha-67.txt &&\
     tar -xjf boost_1_67_0.tar.bz2 &&\
-    rm -rf boost_1_67_0.tar.bz2 boost-sha.txt boost_1_67_0/libs
+    rm -rf boost_1_67_0.tar.bz2 boost-sha-67.txt boost_1_67_0/libs &&\
+    curl -L https://dl.bintray.com/boostorg/release/1.72.0/source/boost_1_72_0.tar.bz2 -o boost_1_72_0.tar.bz2 &&\
+               echo "59c9b274bc451cf91a9ba1dd2c7fdcaf5d60b1b3aa83f2c9fa143417cc660722  boost_1_72_0.tar.bz2" > boost-sha-72.txt &&\
+               sha256sum -c boost-sha-72.txt &&\
+    tar -xjf boost_1_72_0.tar.bz2 &&\
+    rm -rf boost_1_72_0.tar.bz2 boost-sha-72.txt boost_1_72_0/libs
 
 # install cmake
```

I'm not sure why github isn't reporting the conflict.

I was suggesting removing this part of the patch from the pull request.